Octokit.Tests/Clients/PullRequestReviewRequestsClientTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NSubstitute;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class PullRequestReviewRequestsClientTests
+    {
+        public class TheCtor
+        {
+            [Fact]
+            public void EnsuresNonNullArguments()
+            {
+                Assert.Throws<ArgumentNullException>(() => new PullRequestReviewRequestsClient(null));
+            }
+        }
+
+        public class TheGetAlltMethod
+        {
+            [Fact]
+            public async Task RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new PullRequestReviewRequestsClient(connection);
+
+                await client.GetAll("owner", "name", 7);
+
+                connection.Received().GetAll<User>(
+                    Arg.Is<Uri>(u => u.ToString() == "repos/owner/name/pulls/7/requested_reviewers"),
+                    null,
+                    "application/vnd.github.black-cat-preview+json");
+            }
+
+            [Fact]
+            public async Task EnsuresNotNullArguments()
```

Comment:
For consistency, use `EnsuresNonNull` rather than `EnsuresNotNull`.  Despite the fact the former actually sounds better, across the codebase we have 585 cases of NonNull and only 5 of NotNull 🤔